easybuild/easyblocks/l/lammps.py

```diff
@@ -116,19 +117,22 @@ def prepare_step(self, *args, **kwargs):
         super(EB_LAMMPS, self).prepare_step(*args, **kwargs)
 
         # Unset LIBS when using both KOKKOS and CUDA - it will mix lib paths otherwise
-        if self.cfg['kokkos'] and get_software_root('CUDA'):
+        if 'cuda' in [dep['name'].lower() for dep in self.cfg.dependencies()] and get_software_root('CUDA'):
```

Comment:
on second thought, this reads a little funny since it's a boolean you can just do:
`self.cuda = 'cuda' in [dep['name'].lower() for dep in self.cfg.dependencies()] and get_software_root('CUDA')`
Also need to check if `self.cfg.dependencies()` include cuda if the toolchain includes cuda as well, e.g. fosscuda?